docs/conf.py

```diff
@@ -94,6 +94,13 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 html_theme = "bizstyle"
+html_static_path = ['_static']
+
+html_logo = 'logo/hermes_logo.png'
+html_favicon = 'logo/favicon-32x32.png'
```

Comment:
```suggestion
html_favicon = 'logo/favicon.ico'
```
This was using a file `favicon-32x32.png` that does not exist in this PR.